python-modules/chevah-python-test/test_python_binary_dist.py

```diff
@@ -3,106 +3,274 @@
+                'libresolv.so.2',
+                ])
+            rhel_version = int(platform.linux_distribution()[1].split('.')[0])
+            if rhel_version >= 5:
+                expected_deps.extend([
+                    'libkeyutils.so.1',
+                    'libkrb5support.so.0',
+                    'libselinux.so.1',
+                    'libsepol.so.1',
+                ])
+            if rhel_version >= 6:
+                expected_deps.extend([
+                    'libfreebl3.so',
+                ])
+            if rhel_version >= 7:
+                expected_deps.extend([
+                    'liblzma.so.5',
+                    'libpcre.so.1',
+                ])
+    # AIX specific deps.
+    elif platform_system == 'aix':
+        # This is the standard list of deps for AIX 5.3.
+        expected_deps = [
+            '/unix',
+            'libbsd.a',
+            'libc.a',
+            'libcrypt.a',
+            'libcrypto.a',
+            'libdl.a',
+            'libnsl.a',
+            'libpthreads.a',
+            'libpthreads_compat.a',
+            'libssl.a',
+            'libtli.a',
+            'libz.a',
+            ]
+        # sys.platform could be 'aix5', 'aix6' etc.
+        aix_version = int(sys.platform[-1])
+        if aix_version >= 7:
+            expected_deps.extend([
+                'libthread.a',
+            ])
+    # Solaris specific deps.
+    elif platform_system == 'sunos':
+        # This is the standard list of deps for a Solaris 10 build.
+        expected_deps = [
+            'libaio.so.1',
+            'libc.so.1',
+            'libcrypt_i.so.1',
+            'libcrypto.so.0.9.7',
+            'libcrypto_extra.so.0.9.7',
+            'libdl.so.1',
+            'libdoor.so.1',
+            'libgen.so.1',
+            'libintl.so.1',
+            'libm.so.2',
+            'libmd.so.1',
+            'libmp.so.2',
+            'libnsl.so.1',
+            'librt.so.1',
+            'libscf.so.1',
+            'libsocket.so.1',
+            'libsqlite3.so.0',
+            'libssl.so.0.9.7',
+            'libssl_extra.so.0.9.7',
+            'libuutil.so.1',
+            'libz.so.1',
+            ]
+    # OS X specific deps.
+    elif platform_system == 'darwin':
+        # This is the minimum list of deps for OS X.
+        expected_deps = [
+            'ApplicationServices.framework/Versions/A/ApplicationServices',
+            'Carbon.framework/Versions/A/Carbon',
+            'CoreFoundation.framework/Versions/A/CoreFoundation',
+            'CoreServices.framework/Versions/A/CoreServices',
+            'SystemConfiguration.framework/Versions/A/SystemConfiguration',
+            'libSystem.B.dylib',
+            'libcrypto.0.9.8.dylib',
+            'libgcc_s.1.dylib',
+            'libssl.0.9.8.dylib',
+            'libz.1.dylib',
+            ]
+    return expected_deps
+
+
+def get_actual_deps():
+    """
+    Here we get the list of actual deps using a shell script helper.
```

Comment:
Avoid "Here to DO somehting" from doctrings. Keep it simple. Don't add details about implementation.
The docstring is there for the people using this method.
It should state  what it does and the format of the returned value.

"""
Return a list with path of dependent libraries.

Path can be relative or absolute.
"""